yjit/src/backend/x86_64/mod.rs

```diff
@@ -548,6 +549,10 @@ impl Assembler
                     cmovl(cb, insn.out.into(), insn.opnds[1].into());
                 }
                 Op::LiveReg => (), // just a reg alloc signal, no code
+                Op::PadEntryExit => {
+                    let jmp_len = 5; // We assume that our Op::Jmp usage that gets invalidated is <= 5
+                    nop(cb, jmp_len);
```

Comment:
I think we want to use `start_write_pos` here so we're only pad when we need to.